README.md

```diff
@@ -49,6 +49,7 @@ to simulate the `--all` flag.
 
 ## Changelog
 
+* 2.20.0 - Add plugin utilization workflow validator, Fix issue where numbers in screenshot titles would cause validation to fail
```

Comment:
```suggestion
* 2.20.0 - Add plugin utilization workflow validator | Fix issue where numbers in screenshot titles would cause validation to fail
```